jenkins/release.JenkinsFile

```diff
@@ -17,10 +17,10 @@ standardReleasePipelineWithGenericTrigger(
             string(name: 'DOCKER_BUILD_GIT_REPOSITORY', value: 'https://github.com/opensearch-project/opensearch-build'),
             string(name: 'DOCKER_BUILD_GIT_REPOSITORY_REFERENCE', value: 'main'),
             string(name: 'DOCKER_BUILD_SCRIPT_WITH_COMMANDS', value: [
-                'su $(id -un 1000) -c "cd docker/ci',
-                'git clone https://github.com/opensearch-project/opensearch-benchmark opensearch-benchmark',
-                'cp -a opensearch-benchmark/* ./"',
-                'cd docker/ci',
+                "su $(id -un 1000) -c \"cd docker/ci",
+                "git clone https://github.com/opensearch-project/opensearch-benchmark --branch ${tag} --single-branch opensearch-benchmark",
```

Comment:
Our tags are with V.V.V format but our branches use V.V (do not have the patch version). Will this work with the current branch structure still?

Reply:
`$tag` is just a var holding `$ref`: https://github.com/opensearch-project/opensearch-build-libraries/blob/main/vars/standardReleasePipelineWithGenericTrigger.groovy#L58C13-L58C16
And --branch accept reference such as tag name, commit id, branch name as so on despite the param name is `--branch`.

We should be good with this change.

Reply:
Sounds good.  Thanks for clarifying.